theme.json

```diff
@@ -74,22 +74,22 @@
 				},
 				{
 					"name": "Regular",
-					"size": "clamp(30px, 5vw, 50px);",
+					"size": "clamp(30px, 5vw, 50px)",
```

Comment:
The semicolon in `spacingSizes` doesn't seem to cause any warnings, but I'll remove it just to be safe.